modules/reference/pages/properties/broker-properties.adoc

```diff
@@ -135,6 +135,32 @@ TIP: For backward compatibility, `true` is the default. Redpanda recommends usin
 
 ---
 
+=== fips_mode
+
+Controls whether Redpanda starts in FIPS mode.  This property allows for three values: `disabled`, `enabled`, and `permissive`.  
```

Comment:
Suggest making these 3 bullet or definition lists, which will make them easier to read.

eg.
This property allows for the following values:

- `disabled`
Redpanda will not start in FIPS mode.

- `enabled1
Redpanda first verifies that the operating system is enabled for FIPS by checking `/proc/sys/crypto/fips_enabled`.  If the file does not exist or does not return `1`, Redpanda immediately exits.